### PR TITLE
Update UWP C# Voice Assistant README

### PR DESCRIPTION
**Pull request by dargilco**
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:

### CODE REVIEW

File: samples/csharp/uwp/virtualassistant-uwp/README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
-# Quickstart: Using virtual assistant (Preview) in C# Universal Windows Platform
+# Quickstart: Using Voice Assistant in C# Universal Windows Platform
```
Comment (trrwilson) [inline samples/csharp/uwp/virtualassistant-uwp/README.md:1]: I'd recommend the plural of "Voice Assistants" for the switch

File: samples/csharp/uwp/virtualassistant-uwp/README.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 
-This sample demonstrates how to recognize speech for custom voice-first virtual assistants (Preview) and receive activity responses.
+This sample demonstrates how to recognize speech for Voice Assistant and receive activity responses.
```
Comment (trrwilson) [inline samples/csharp/uwp/virtualassistant-uwp/README.md:3]: Recommend replacing the line with
```
This sample demonstrates how to use Voice Assistants to recognize speech and receive activity responses.
```

File: samples/csharp/uwp/virtualassistant-uwp/README.md (hunk 1 above)
Comment (dargilco) [reply samples/csharp/uwp/virtualassistant-uwp/README.md:1]: Sure!

**Review by trrwilson — APPROVED** [review]

File: samples/csharp/uwp/virtualassistant-uwp/README.md (hunk 2 above)
Comment (dargilco) [reply samples/csharp/uwp/virtualassistant-uwp/README.md:3]: Yeah that sounds better, thanks!

**Review by trrwilson — APPROVED** [review]

**Pull request merged**